src/main/java/org/prebid/server/spring/config/SettingsConfiguration.java

```diff
@@ -134,21 +134,25 @@ private static BasicJdbcClient createBasicJdbcClient(
 
         @Bean
         JDBCClient vertxJdbcClient(Vertx vertx, StoredRequestsDatabaseProperties storedRequestsDatabaseProperties) {
+            final boolean isHikari = storedRequestsDatabaseProperties.getProviderClass() == DbPoolType.hikari;
```

Comment:
Use switch for properties resolving, can be useful when we will have more than two values